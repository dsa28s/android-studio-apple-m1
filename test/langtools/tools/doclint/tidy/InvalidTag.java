/*
 * @test /nodynamiccopyright/
 * @bug 8004832
 * @summary Add new doclint package
 * @library ..
 * @modules jdk.compiler/com.sun.tools.doclint
 * @build DocLintTester
 * @run main DocLintTester -ref InvalidTag.out InvalidTag.java
 */

// tidy: Error: <.*> is not recognized!

/**
 * List<String> list = new ArrayList<>();
 */
public class InvalidTag { }
