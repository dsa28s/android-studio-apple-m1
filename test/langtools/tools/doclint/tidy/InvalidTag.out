InvalidTag.java:14: error: unknown tag: String
 * List<String> list = new ArrayList<>();
       ^
InvalidTag.java:14: error: malformed HTML
 * List<String> list = new ArrayList<>();
                                    ^
InvalidTag.java:14: error: bad use of '>'
 * List<String> list = new ArrayList<>();
                                     ^
3 errors
